package.json

```diff
@@ -35,9 +35,11 @@
     "serve": "npm-run-all build:assets build:site -p eleventy:serve watch:js watch:sass",
     "build:site": "run-s site:clean sassdoc",
     "build:assets": "npm-run-all assets:clean -p js sass",
-    "build": "run-s build:assets build:site eleventy"
+    "build": "run-s build:assets build:site eleventy",
+    "test": "jest"
   },
   "devDependencies": {
+    "#": "link:./src/filters",
```

Comment:
@duggiemitchell @mirisuzanne I added this link so we can import or require from e.g. `#/pages` instead of using the full path.

Reply:
Why `#` rather than  `~`?

Reply:
Because it's specifically a shortcut to the 11ty filters, not to node modules -- I didn't want to confuse the two. 🤷‍♂ 

This is all arbitrary, of course. We could just use a generic shortcut to `src/` and do e.g. `require('#/filters/pages')` if we want.